src/components/TODOList.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Todo } from '../templates/Practice5';
+
+type TODOListProps = {
+  todos: Todo[];
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+};
+
+export const TODOList = ({ todos, setTodos }: TODOListProps) => {
+  const [allChecked, setAllChecked] = useState<boolean>(false);
```

Comment:
To Follow our [React guidelines ](https://pj100.esa.io/posts/16722#%E3%82%A2%E3%83%B3%E3%83%81%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3%E3%81%AE%E6%94%B9%E5%96%84)

```suggestion
  const [isAllChecked, setIsAllChecked] = useState<boolean>(false);
```

Reply:
Changed state name to isAllChecked